mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -280,6 +280,48 @@ class PythonMLLibAPI extends Serializable {
   }
 
   /**
+   * Java stub for Python mllib GaussianMixtureEM.train()
+   */
+  def trainGaussianMixtureEM(
+      data: JavaRDD[Vector], 
+      k: Int, 
+      convergenceTol: Double, 
+      seed: Long, 
+      maxIterations: Int): JList[Object]  = {
+    val gmmAlg = new GaussianMixtureEM()
+      .setK(k)
+      .setConvergenceTol(convergenceTol)
+      .setSeed(seed)
+      .setMaxIterations(maxIterations)
+    try {
+      val model = gmmAlg.run(data.rdd.persist(StorageLevel.MEMORY_AND_DISK))
+      List(model.weight, model.mu, model.sigma).
+      map(_.asInstanceOf[Object]).asJava
+    } finally {
+      data.rdd.unpersist(blocking = false)
+    }
+  }
+
+  /**
+   * Java stub for Python mllib GaussianMixtureModel.predictSoft()
+   */
+  def findPredict(
```

Comment:
This is inside `PythonMLlibAPI`. It is necessary to mention GMM in the method name. Btw, I don't quite understand what `find` means here.